pkg/apis/playlist/v1/handler.go

```diff
@@ -47,54 +47,83 @@ func (r *handler) ConvertToTable(ctx context.Context, object runtime.Object, tab
 
 func (r *handler) List(ctx context.Context, options *internalversion.ListOptions) (runtime.Object, error) {
 	ns, ok := request.NamespaceFrom(ctx)
-	if ok && ns != "" {
-		orgId, err := apis.NamespaceToOrgID(ns)
-		if err != nil {
-			return nil, err
-		}
-		fmt.Printf("OrgID: %d\n", orgId)
+	if !ok || ns == "" {
+		return nil, fmt.Errorf("namespace required")
```

Comment:
Not really related to this PR, but how does this work outside of cloud? do we use a generic namespace for single-tenant deployment?

Reply:
"default" is orgId=1, that seems easier than removing namespacing entirely for single-tenant deployments.  Essentially it adds "namespaces/default/" to a bunch of URLs when that is not really necessary (but makes the patterns consistent for single or multi-tenant)

Reply:
interesting, in that case we will have to check the orgs the user can see and update the query